atom/browser/api/lib/web-contents.coffee

```diff
@@ -34,6 +34,12 @@ module.exports.wrap = (webContents) ->
     do (name, method) ->
       webContents[name] = -> method.apply controller, arguments
 
+  # Service-worker utilities.
+  webContents.hasServiceWorker = (callback) ->
+    @_hasSW @getUrl(), callback
```

Comment:
Havnt we moved the getURL implementation to JS with the new navigationController ? Thats why i went with overriding the api on JS.